app/models/metric/rollup.rb

```diff
@@ -127,6 +127,21 @@ module Metric::Rollup
 
   TIMEOUT_PROCESS = 30.minutes.to_i
   DERIVED_COLS_EXCLUDED_CLASSES = ['MiqRegion', 'MiqEnterprise']
+  STAT_COLS_EXCLUDED_CLASSES = [
+      'MiqRegion',
+      'MiqEnterprise',
+      'VmOrTemplate',
+      'Container',
+      'ContainerGroup',
+      'ContainerNode',
+      'ContainerProject',
+      'ContainerReplicator',
+      'ContainerService',
+      'Host',
+      'AvailabilityZone',
+      'EmsCluster',
+      'MiqRegion',
+      'MiqEnterprise']
```

Comment:
@yaacov what if anyone is adding new classes? Will he always need to add it here?

It seems much easier to go with the opposite logic of `STAT_COLS_CLASSES` (to mention the single class for now that has stats).